src/unstract/adapters/llm/ollama/src/ollama.py

```diff
@@ -62,6 +62,7 @@ def get_llm_instance(self) -> LLM:
                 ),
                 json_mode=False,
                 context_window=int(self.config.get(Constants.CONTEXT_WINDOW, 3900)),
+                temperature=0.01,
```

Comment:
How does this matter @harini-venkataraman ?

Reply:
@gaya3-zipstack This is regarding ollama temperature set as 0.75 by default. Since @harini-venkataraman was making change in adapters, I asked her to include this change as well.

Reply:
@gaya3-zipstack Yes, the temperature in options was sent with default value. @Deepak-Kesavan suggested to include in this PR.